packages/server/src/client/internal/open-feature-client.ts

```diff
@@ -223,6 +224,19 @@ export class OpenFeatureClient implements Client {
     return this.evaluate<T>(flagKey, this._provider.resolveObjectEvaluation, defaultValue, 'object', context, options);
   }
 
+  track(occurrenceKey: string, context: EvaluationContext, occurrenceDetails: TrackingEventDetails): void {
+
+    this.shortCircuitIfNotReady();
+
+    const mergedContext = this.mergeContexts(context);
+
+    if (typeof this._provider.track === 'function') {
+      return this._provider.track?.(occurrenceKey, mergedContext, occurrenceDetails);
```

Comment:
Shouldn't we log and swallow any exceptions thrown by the provider?

Reply:
Definitely; this would fall into our spec rules for client ops not abnormally terminating. I will add some error handling.

Reply:
Added